ui/src/app/dashboard/grouped-summary/grouped-summary.component.html

```diff
@@ -1,31 +1,45 @@
-<mat-card>
-  <mat-card-title>
-    {{ aggregation.name }}
-  </mat-card-title>
+<div [class]="cardClass" id="container">
+  <mat-card>
+    <mat-card-content>
+      <table matSort (matSortChange)="sortData($event)">
+        <tr>
+          <th mat-sort-header="label" class="label">{{ aggregation.name }}</th>
+          <ng-container *ngFor="let status of statusArray">
+            <th [mat-sort-header]=status>
```

Comment:
Doesn't have to be in this PR, but similar to the list page we should have tooltips explaining what the status icons mean